index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+<meta charset=“UTF-8”>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="reset.css">
+<link rel="stylesheet" href="style.css">
+<title>Lista zakupów</title>
+</head>
+
+<body>
+<div class="top_box">
+	<div class="logo">
+	<p>Lista zakupów</p>
+	</div>
+</div>
+
+<div class="main_box">
+
+	<div class="category_box">
+	
+		<div class="category_name">
+			<p>Alkohole</p>
+		</div>
+		
+		<div class="list_box">
+
+			<ul class="upper_list">
+				<li>Whisky</li>
+				<li>Likier</li>
+				<li>Gin</li>
+				<li>Wino</li>
+			</ul>
+		
+		</div>
+		<div class="confirmation_box">
+			<label> Kupione:
+			<input type="checkbox">
```

Comment:
tutaj fajnie by było dodać name, id oraz do `labela` atrybut for z idkiem tego inputa